src/SignalRServiceExtension/Bindings/SignalRCollectorBuilder.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Text;
+
+namespace Microsoft.Azure.WebJobs.Extensions.SignalRService
+{
+    internal class SignalRCollectorBuilder<T> : IConverter<SignalRAttribute, IAsyncCollector<T>>
+    {
+        private readonly SignalRConfigProvider _configProvider;
```

Comment:
Nit: privates in the rest of the codebase don't use `_`. If we want to switch all to this convention, perhaps do it in another commit/PR.

Reply:
OK. Will uniform styles and might do this in the future.